service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ResourceTypeCodes.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class ResourceTypeCodes {
+
+    public static final String CONSULTATION_LIST_CODE = "325851000000107";
+    public static final String OTHER_REPORT_CODE = "24591000000103";
+    public static final String TOPIC_LIST_CODE = "25851000000105";
+    public static final String CATEGORY_LIST_CODE = "24781000000107";
+    public static final String BLOOD_PRESSURE_READING_CODE = "163020007";
+    public static final String ARTERIAL_BLOOD_PRESSURE_CODE = "386534000";
+    public static final String BLOOD_PRESSURE_CODE = "75367002";
+    public static final String STANDING_BLOOD_PRESSURE_CODE = "163034007";
+    public static final String SITTING_BLOOD_PRESSURE_CODE = "163035008";
+    public static final String LAYING_BLOOD_PRESSURE_CODE = "163033001";
+    public static final String ENCOUNTER_LIST_CODE = "1149501000000101";
+    public static final String NARRATIVE_STATEMENT_CODE = "37331000000100";
```

Comment:
`NARRATIVE_STATEMENT_CODE` is the same as `COMMENT_NOTE_CODE`
I know it's the same as before your change but can you confirm this is ok?

Reply:
Both codes are used for checking if the observation is a narrative. I'll remove one of them, then.

Reply:
done